test/Engine/ProtoTest/ProtoTest.csproj

```diff
@@ -174,6 +174,10 @@
     <Compile Include="UtilsTests\TextFxTests.cs" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\..\src\DynamoUtilities\DynamoUtilities.csproj">
+      <Project>{b5f435cb-0d8a-40b1-a4f7-5ecb3ce792a9}</Project>
+      <Name>DynamoUtilities</Name>
+    </ProjectReference>
```

Comment:
What's this needed for?

Reply:
It is needed for the AssemblyHelper class
I had to use it in a couple of places to be able to test out loading of assemblies from the /nodes folder